org.freecadweb.FreeCAD.yaml

```
@@ -610,6 +610,21 @@ modules:
   - python3-pyyaml.json
   - python3-Pillow.json
 
+  # pybind11 used by mesh flattening feature
+  - name: pybind11
+    buildsystem: cmake-ninja
+    builddir: true
+    build-options:
+      - --target=pybind11::headers
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DPYBIND11_INSTALL=ON
+      - -DPYBIND11_TEST=OFF
+    sources:
+      - type: archive
+        url: http://deb.debian.org/debian/pool/main/p/pybind11/pybind11_2.9.0.orig.tar.gz
```

```suggestion
        url: https://deb.debian.org/debian/pool/main/p/pybind11/pybind11_2.9.0.orig.tar.gz
```

at least let's get it over https.

---

done ;)